docs/release-engineering.md

```diff
@@ -7,6 +7,44 @@ title: How to build and ship a release
 
 Here are my notes from producing the v0.2.8 release.  Change the version number as appropriate.
 
+## Tip: How to keep modules updated
+
+List out-of-date modules and update any that 
+
+```
+go get -u github.com/psampaz/go-mod-outdated
+go list -u -m -json all | go-mod-outdated -update -direct 
+```
+
+To update a module, `get` it, then re-run the unit and integration tests.
+
+```
+go get -u
+    or
+go get -u module/path
+```
+
+Once the updates are complete, tidy up:
+
+```
+go mod tidy
+```
+
+When done, vendor the modules (see Step 0 below).
+
+## Step 0. Vendor the modules
+
+Vendor the modules. The vendored files are not used (unless you change
```

Comment:
For future reference, I'd also probably mention that they are loaded from the module cache instead, and that unlike using vendor, they are verified.